extensions/amp-analytics/0.1/test/vendor-requests.json

```diff
@@ -18,6 +18,11 @@
     "interval": "https://ping.chartbeat.net/ping?h=$domain&p=_canonical_path_&u=_client_id_&d=_canonical_host_&g=$uid&g0=$sections&g1=$authors&g2=$zone&g3=$sponsorName&g4=$contentType&c=120&x=_scroll_top_&y=_scroll_height_&j=$decayTime&R=1&W=0&I=0&E=_total_engaged_time_&r=_document_referrer_&t=_page_view_id__client_id_&b=_page_load_time_&i=_title_&T=_timestamp_&tz=_timezone_&C=2&_",
     "anchorClick": "https://ping.chartbeat.net/ping?h=$domain&p=_canonical_path_&u=_client_id_&d=_canonical_host_&g=$uid&g0=$sections&g1=$authors&g2=$zone&g3=$sponsorName&g4=$contentType&c=120&x=_scroll_top_&y=_scroll_height_&j=$decayTime&R=1&W=0&I=0&E=_total_engaged_time_&r=_document_referrer_&t=_page_view_id__client_id_&b=_page_load_time_&i=_title_&T=_timestamp_&tz=_timezone_&C=2&_"
   },
+  "colanalytics": {
+    "host": "https://ase.clmbtech.com",
```

Comment:
I am not sure how the tests pass.  Where does the value ase.clmbtech.co come from?